cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -43,6 +46,47 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
       );
     });
 
+    if (
+      serviceHasInlineLink(service) &&
+      (Cypress.env('APP_ENV') === 'local' || Cypress.env('APP_ENV') === 'test')
```

Comment:
Why is this limited to some environments?

Reply:
Reason being the article on live which we are using in our tests doesn't have an inline link....could ask @jimjohnsonrollings whether we could update the live article.

Reply:
Thanks, I think that approach is better than adding lots of extra unnecessary conditions to the tests.

Reply:
All the articles we are using to test should have an inline link to the BBC Weather homepage.